sgx-traits/src/lib.rs

```diff
@@ -16,7 +16,12 @@ use std::io::Result;
 
 use bitflags::bitflags;
 use sgx_types::page::{Flags as PageFlags, SecInfo};
-use sgx_types::{secs::Secs, sig::Signature, tcs::Tcs, Offset};
+use sgx_types::{
+    secs::Secs,
+    sig::Signature,
+    tcs::Tcs,
+    Offset,
+};
```

Comment:
What is the purpose of this change?

Reply:
It's `cargo fmt`. In general I'm still not sure if `cargo fmt` changes should be their own PR... that seems sort of cluttered for the `git log`? But otherwise it ends up getting folded in, like this. Open to suggestions.

Reply:
The question is why is `cargo fmt` making this change? The previous commits had `cargo fmt` run on them. So we shouldn't be seeing this.

Reply:
I'll merge this PR and figure it out afterwards.

Reply:
I just ran it again and it went away, so I'm confused about it too. Let me fix it and I'll re-push.